### PR TITLE
[MM-29167] Add e2e tests for searchTeamCmdF

### PR DESCRIPTION
**Pull request by sirMackk**
Adds e2e tests for the `searchTeamCmdF` function.

#### Ticket Link

Fixes https://github.com/mattermost/mattermost-server/issues/15673

JIRA: https://mattermost.atlassian.net/browse/MM-29167

### CODE REVIEW

File: commands/team_e2e_test.go [hunk 1]
```diff
@@ -0,0 +1,47 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package commands
+
+import (
+	"github.com/mattermost/mattermost-server/v5/model"
+	"github.com/spf13/cobra"
```
Comment (agnivade) [inline commands/team_e2e_test.go:8]: Typically we keep 3rd party imports in a separate group. Let's do that here too.

**Review by agnivade — COMMENTED** [review]
Thanks @sirMackk ! Just a small comment, looks good otherwise.

File: commands/team_e2e_test.go (hunk 1 above)
Comment (sirMackk) [reply commands/team_e2e_test.go]: Fixed. Thanks for catching that!

**Review by agnivade — APPROVED** [review]
Thanks!

**Review by mgdelacroix — APPROVED** [review]
LGTM!! Thanks for the contribution @sirMackk!! :rocket:

**Comment by sirMackk** [comment]
Rebased and resolved conflicts.

**Review by lindy65 — APPROVED** [review]
Thanks @sirMackk @mgdelacroix - looks good to me 👍

**Pull request merged**